_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,112 @@
+title: Banner
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey meaning in the banner.
+    summary_additional: When viewing the banner, you never have to rely on color alone to understand meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Banner appears consistently.
+    summary_additional: The banner appears in the same location on every page of your government website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Icons meet color contrast requirements.
+    summary_additional: |
+       When you open the banner and use ANDI or another color contrast analyzer to look at the hex numbers,
+       contrast between the icons and the background is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification tests
+  - summary: Text retains functionality when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%, text in the banner remains visible and retains all of its functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Banner content is visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 400%,
+      banner content adjusts to fit the width of the screen without losing functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Banner can be accessed, opened, and closed with only a keyboard.
+    summary_additional: |
+      When you navigate your site with a keyboard,
+      you can access the banner by using `tab` to move forward or `shift` + `tab` to move backwards.
+      You can use the `enter` and `spacebar` keys to open and close the banner.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: Banner does not trap keyboard focus.
+    summary_additional: |
+      When you navigate with a keyboard, you can easily move into and out of the banner,
+      regardless of whether the banner is open or closed.
+      When you navigate out of an open banner using your keyboard, it doesn't automatically close.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Banner is the first interactive object after the main navigation.
+    summary_additional: |
+      When you navigate your site with a keyboard and skip navigation,
+      the "Here's how you know" banner button is the first focusable item on the page.
```

Comment:
This is for focus order. https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

Also I think it needs to be reworded a bit:
The Banner button is the first interactive object after the "skip navigation" link.  When you navigate your site with a keyboard and bypass the "skip navigation" link, the “Here’s how you know” banner button is the first focusable item on the page.

Reply:
Additionally, I think this may NOT be the case if a site alert is implemented. I think the [site alert ](https://designsystem.digital.gov/components/site-alert/)might be read aloud first AFTER the skip nav link. So the focus order would go:

1. Skip to main content
2. Site alert (if there are focusable items in the alert). Site alerts would be announced first to AT users, however if marked as "alerts".
3. Banner "here's how you know" button

Reply:
@amycole501 I updated the summary to match your suggestion in [390328f](https://github.com/uswds/uswds-site/pull/2687/commits/390328fb7edbaeec041978c6135abef062303ce0). Do we need to re-write the summary on this to account for the site alert exception, or leave it as is? 

@sarah-sch Do you want to proof the updated summary here?

Reply:
@amyleadem - the update looks good.